docs/tutorials/python/sagas/index.md

```diff
@@ -0,0 +1,551 @@
+
+Create a new file named `run_workflow.py`:
+
+First, import the necessary modules, including `uuid`, Flask, and Temporal `Client`.
+
+```python
+import uuid
+
+from flask import Flask, jsonify, request
+from temporalio.client import Client
+
+from activities import BookVacationInput
+from saga_workflows import BookWorkflow
+from shared import TASK_QUEUE_NAME
+```
+
+Initialize the Flask app and set up the dependency injection for the Temporal client.
+
+```python
+app = Flask(__name__)
+
+
+async def get_temporal_client():
+    return await Client.connect("localhost:7233")
+```
+
+Define a route to handle the booking process.
+This route will accept a `POST` request, extract the necessary data from the request, initiate the Workflow, and return the result.
+
+```python
+@app.route("/book", methods=["POST"])
+async def book_vacation():
+    # Extract data from the request
+    user_id = f'{request.json.get("name").replace(" ", "-").lower()}-{str(uuid.uuid4().int)[:6]}'
+    attempts = request.json.get("attempts")
+    car = request.json.get("car")
+    hotel = request.json.get("hotel")
+    flight = request.json.get("flight")
+
+    # Create the input object for the Workflow
+    input = BookVacationInput(
+        attempts=int(attempts),
+        book_user_id=user_id,
+        book_car_id=car,
+        book_hotel_id=hotel,
+        book_flight_id=flight,
+    )
+
+    # Get the Temporal client
+    client = await get_temporal_client()
+
+    # Execute the Workflow
+    result = await client.execute_workflow(
+        BookWorkflow.run,
+        input,
+        id=user_id,
+        task_queue=TASK_QUEUE_NAME,
+    )
+
+    # Prepare the response based on the result
+    response = {"user_id": user_id, "result": result}
+    if result == "Voyage cancelled":
+        response["cancelled"] = True
+    else:
+        result_list = result.split("Booked ")
+        response["cancelled"] = False
+        response["car"] = result_list[1].split(" Booking ")[0].title()
+        response["hotel"] = result_list[2].split(" Booking ")[0].title()
+        response["flight"] = result_list[2].split(" Booking ")[1].title()
+
+    return jsonify(response)
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", debug=True)
+```
+
+The route extracts the username, number of attempts, car, hotel, and flight information from the request JSON.
+
+A `BookVacationInput` object is created with the extracted data, which will be passed to the Workflow.
+
+The Temporal client is obtained using the `get_temporal_client()` function.
+
+The Workflow is executed using `client.execute_workflow()`, passing the input object and other required parameters.
+Based on the result of the Workflow execution, a response is prepared and returned.
+If the booking process is cancelled, the response indicates this. Otherwise, it provides details about the booked car, hotel, and flight.
+
+**Start the Client**
+
+Now to start the Client, run the following command in your terminal:
```

Comment:
Don't I already have my terminal blocked with a worker?

We need to tell people to open a new window, switch to the project dir, and then do the start process.